modules/gltf_blend/editor_scene_importer_blend.cpp

```diff
@@ -0,0 +1,272 @@
+/*************************************************************************/
+/*  editor_scene_importer_blend.cpp                                      */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2022 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2022 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#if TOOLS_ENABLED
+#include "editor_scene_importer_blend.h"
+
+#include "core/config/project_settings.h"
+#include "core/io/json.h"
+#include "core/string/print_string.h"
+#include "editor/editor_settings.h"
+#include "modules/gltf/editor_scene_importer_gltf.h"
+#include "modules/gltf/gltf_document.h"
```

Comment:
I would suggest moving this code to the `gltf` module directly, so you don't need to have cross-module dependencies like this. If it's only usable with gltf, and doesn't add a lot of code or dependencies, it can all be in the same module.

Reply:
For code organization purposes I'd like it to be separate if possible.

Reply:
Don't have a strong opinion. The same structure is used for FBX. There's a lot of code in registration.

Reply:
It's a single file, and it depends on the gltf module, so I still think it should go in `modules/gltf/editor/`.
The same will apply for the FBX to glTF importer. It's cleaner IMO if all glTF features are in the `gltf` module.

It can also also be in `modules/gltf/editor/blend/` if you want it in its own folder.

Reply:
Done in https://github.com/akien-mga/godot/tree/blend-io